integrationTests/cas-browser.spec.js

```diff
@@ -0,0 +1,105 @@
+const config = require('./config');
+const utils = require('./utils');
+const webdriver = require('selenium-webdriver');
+const By = webdriver.By;
+const until = webdriver.until;
+const waitInterval = 2000;
+
+const logoutUrl = '/cas/logout';
+const loginUrl = '/cas/login';
+
+
+jest.setTimeout(30000);
+
+let driver;
+
+beforeEach(() => {
+    driver = utils.createDriver(webdriver);
+});
+
+afterEach(() => {
+    driver.quit();
+});
+
+describe('cas browser login', () => {
+
+    test('automatic redirect to cas login', async () => {
+        await driver.get(config.baseUrl + config.nexusContextPath);
+        const url = await driver.getCurrentUrl();
+        expect(url).toMatch(loginUrl);
+    });
+
+    test('login', async() => {
+        await driver.get(utils.getCasUrl(driver));
+        await utils.login(driver);
+        await driver.sleep(waitInterval)
+        // find user account button holding the username
+        const username = await driver.findElement(By.id('button-1142-btnInnerEl')).getText();
+        expect(username.toLowerCase()).toContain(config.displayName);
```

Comment:
Why are you using 'toContain'? Doesn't it allow names containing the username as well (like 'adminhallo' would be correct even if the user's name is only 'admin')?